Framework7/features/feature_definitions/get_comment_feature/get_comment

```diff
@@ -0,0 +1,8 @@
+@getting_comments
```

Comment:
Thanks for your corrections i am going to fix them.
Bye, take care.
El 26/5/2016 7:28, "wMercadoJ" <notifications@github.com> escribió:

> In Framework7/features/feature_definitions/get_comment_feature/get_comment
> <https://github.com/wMercadoJ/QA-DEV07/pull/12#discussion_r64730146>:
>
> > @@ -0,0 +1,8 @@
> > +@getting_comments
>
> This file should be *.feature. Right?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wMercadoJ/QA-DEV07/pull/12/files/7a8a38bebeced5fb0b4d9a8242615b64917bf935#r64730146>
>